Observer/NewsletterSubscribeObserver.php

```diff
@@ -7,21 +7,21 @@
 
 class NewsletterSubscribeObserver implements ObserverInterface
 {
-    protected $data_helper;
+    protected $_dataHelper;
```

Comment:
this actually isn't a convention in php since you have the ability to set visibility.

Reply:
ahhh, I guess never mind. probably better to just be consistent since we're already using that convention https://github.com/klaviyo/magento2-klaviyo/pull/54/files#diff-735f70c1f948ebbd90915b1548d93432R12